localization/english/names/wc_character_names_l_english.yml

```
@@ -15860,6 +15886,1447 @@
  Zihyx:0 "Zihyx"
  Zilzith:0 "Zilzith"
  Grizzik:0 "Grizzik"
+ Brith:0 "Brith"
+ Chal:0 "Chal"
+ Desh:0 "Desh"
+ Fad:0 "Fad"
+ Gresh:0 "Gresh"
+ Kik:0 "Kik"
+ Makk:0 "Makk"
+ Meth:0 "Meth"
+ Mex:0 "Mex"
+ Mok:0 "Mok"
+ Pam:0 "Pam"
+ Pih:0 "Pih"
+ Por:0 "Por"
+ Reth:0 "Reth"
+ Ril:0 "Ril"
+ Skak:0 "Skak"
+ Srash:0 "Srash"
+ Tah:0 "Tah"
+ Vesh:0 "Vesh"
+ Thaurissan:0 "Thaurissan"
+ Roka:0 'Ro'ka'
```

```suggestion
 Roka:0 "Ro'ka"
```
These are reported as an error.
```
[localization_reader.cpp:184]: Expected quotation mark (") at line 15909 and column 9 in localization/english/names/wc_character_names_l_english.yml
```